### ISSUE TITLE
dials command line programs: report parameters by default

### ISSUE THREAD

**Issue by rjgildea** [dials/dials#311]
The current method of requiring (e.g.) '-c -e1 -a1' is rather arcane and not obvious to most users. I think it would be better to show at least the main parameters (and probably help strings) by default, along with the program description. Compare to (e.g.) phenix.xtriage output:

```
$ phenix.xtriage
Usage: 
phenix.xtriage [options] reflection_file parameters [...]

Full parameters:

  scaling {
    input {
      asu_contents
        .help = "Defines the ASU contents"
      {
        sequence_file = None
          .help = "File containing protein or nucleic acid sequences. Values"
                  "for n_residues and n_bases will be extracted automatically"
                  "if this is provided."
        n_residues = None
          .help = "Number of residues in structural unit"
        n_bases = None
          .help = "Number of nucleotides in structural unit"
        n_copies_per_asu = None
          .help = "Number of copies per ASU. If not specified, Matthews"
                  "analyses is performed"
      }
...
```

**Comment by jmp1985** [comment]
This comes up periodically. I'm happy to do whatever most people think is best but last time it was discussed I think we decided that showing a small help and then doing "-c" to show the full configuration was better than showing parameters by default. I guess if we want to do that then we should probably have another look at what parameters should be under what expert level.

________________________________

**Comment by luisodls** [comment]
If we are going to work on number of parameters and expert levels, I would like to highlight that we should reduce the beginner level to 5 or less parameters. In today's version of the GUI I have hard-coded a small subset of parameters that is NOT the same as in the dials command line that is about to run.

L
________________________________

**Comment by Anthchirp** [comment]
autogenerate man pages? :)

**Comment by dagewa** [comment]
I believe this was fixed by #1920
